oe_paragraphs.post_update.php

```diff
@@ -125,3 +126,57 @@ function oe_paragraphs_post_update_10004(array &$sandbox): void {
     $display->save();
   }
 }
+
+/**
+ * Installs Banner paragraph.
+ */
+function oe_paragraphs_post_update_10005(&$sandbox): void {
+  // Install required maxlength module.
+  \Drupal::service('module_installer')->install(['maxlength']);
+
+  // Add paragraph form mode variants.
+  $form_modes = [
+    'paragraph.oe_banner_image' => 'Image banner',
+    'paragraph.oe_banner_image_shade' => 'Image shade banner',
+    'paragraph.oe_banner_primary' => 'Primary banner',
+  ];
+  foreach ($form_modes as $id => $label) {
+    $form_mode = EntityFormMode::create([
+      'id' => $id,
+      'label' => $label,
+      'targetEntityType' => 'paragraph',
+    ]);
+    $form_mode->save();
+  }
+
+  // Add Banner paragraph.
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs') . '/config/post_updates/10005');
+
+  \Drupal::entityTypeManager()->getStorage('paragraphs_type')
+    ->create($storage->read('paragraphs.paragraphs_type.oe_banner'))
+    ->save();
+
+  $field_config = [
```

Comment:
I know we name this variable the same in a previous method but this are not field configurations anymore, there is also field storage configuration, form and view display configurations and so on. I think I'd rather call this $config_ids.